deploy-options.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+# If enabled, console will be cleared on every script run.
+# By default, this option is enabled
+use_automatic_console_clean=true
+# If enabled, not pushed commits will be pushed automatically without confirmation dialog.
+# By default, this option is disabled
+enable_automatic_commit_push=false
+# If enabled, confirmation dialog with deploy summary will be presented.
+# By default, this option is enabled
+enable_final_confirmation=true
+
+#########################################################
+#                     DEPLOY OPTIONS                    #
+#                                                       #
+# Part of script that should be edited by the user.     #
+#                                                       #
+# You can add/remove/edit targets that you want to use  #
+# for the deployment.                                   #
+#                                                       #
+# This will generate first part of tag.                 #
+#########################################################
+
+## Used for the option select by the user
+## And for generating first part of the tag.
+## If option is added to the "Select target"
+## same option should be added under the 
+## if/else logic for creating the first part.
+
+## WARNING: This file should be in the same folder as `app-deploy.sh` file
+
+function deploy_options {
+
+    # Options shown to the user
+
+    echo
+    echo "###############################################################"
+    echo "#                  DEPLOY TARGET SELECTION                    #"
+    echo "###############################################################"
+    echo
+    echo "--------------"
+    echo "| TryOutApps |"
+    echo "--------------"
+    echo
+    echo "[0] All"
+    echo "[1] Staging"
+    echo "[2] UAT"
+    echo "[3] Production"
+    echo
+    echo "=================="
+    echo
+    echo "---------------------"
+    echo "| APP STORE CONNECT |"
+    echo "---------------------"
+    echo
+    echo "[4] App Store"
+    echo
+    read -r -p "Enter number in square brackets: " target_selection
```

Comment:
I think @z-turk3 added support for adding multiple numbers as input, so that's something that could be cool, or added in next version of the script

Reply:
We could add this in the next update! :)